Screenbox/Strings/en-US/Resources.resw

```diff
@@ -803,20 +824,26 @@
   <data name="GetLivelyVisuals" xml:space="preserve">
     <value>Download visuals on GitHub</value>
   </data>
+  <data name="VisualizerOptionsToolTip" xml:space="preserve">
+    <value>Audio visual options</value>
+  </data>
+  <data name="Visualizers" xml:space="preserve">
+    <value>Visuals</value>
+  </data>
   <data name="BrowseFiles" xml:space="preserve">
-    <value>Browse file(s)</value>
+    <value>Browse files</value>
```

Comment:
Why change this? This style is consistent with the inbox media player.

Reply:
Same as above, also it's better for screen readers and consistent with the System Settings

Reply:
I would use "Browse files"